src/test/scala/com/ubirch/webui/models/keycloak/DevicesSpec.scala

```diff
@@ -396,9 +396,60 @@ class DevicesSpec extends FeatureSpec with EmbeddedKeycloakUtil with Matchers wi
       println(deviceClaimed.toDeviceFE().toString)
     }
 
+    scenario("add one device and claim it group password") {
+
+      val builderResponse = TestRefUtil.initKeycloakDeviceUser(initKeycloakBuilderNoDevice)
```

Comment:
same here, very long and unreadable scenario ;)

Reply:
I really need to focus on the clarity of the tests.. For another PR ;)